### DISCUSSION TITLE
Connection link in Rabbitmq console return HTTP400

### DISCUSSION THREAD

**Discussion by sarditi** [rabbitmq/rabbitmq-server#6123] [Other]
Hello,
As depicted in the attached, once a link is pressed nothing happen and an HTTP 400 error occurs.
No message logs in the server.
Thanks
Sagi

![image](https://user-images.githubusercontent.com/1714576/195825513-8c0a7ee2-0b9a-47d6-9b63-9e139a58c6b8.png)

**Comment by michaelklishin** [comment]
We cannot suggest much with a single screenshot. 

HTTP API requests are not [logged unless you opt in](https://www.rabbitmq.com/management.html#http-logging). There may be intermediaries (proxies, load balancers) that can respond before the request even gets to RabbitMQ.

This is not a commonly reported behavior, so there is something environment-specific at play.
